index.html

```diff
@@ -1393,9 +1396,17 @@ <h3>
           following algorithm. The algorithm takes a <a>string</a>
           <var>text</var> as an argument, which represents a <a>manifest</a>,
           and a <a>URL</a> <var>manifest URL</var>, which represents the
-          location of the manifest, and a <a>URL</a> <var>document URL</var>.
-          The output from inputting an JSON document into this algorithm is a
-          <dfn>processed manifest</dfn>.
+          location of the manifest, and an optional <a>URL</a> <var>document
+          URL</var>. The output from inputting an JSON document into this
+          algorithm is a <dfn>processed manifest</dfn>.
+        </p>
+        <p class="issue">
+          There is currently no way to invoke this algorithm without a
+          <var>document URL</var>. However, several user agents do just that
```

Comment:
This doesn't make sense: the previous paragraph says document URL is optional when invoking this algorithm, but this implies it's not optional.

Perhaps you mean that everything that invokes this algorithm requires a document URL, and the actual issue should be "All invocations of this algorithm currently supply a document URL, but several user agents invoke this etc."

Reply:
Fixed (and moved).